rc/filetype/org.kak

```diff
@@ -0,0 +1,262 @@
+add-highlighter shared/org/src_block region ^(\h*)(?i)#\+BEGIN_SRC(\h+|\n) ^(\h*)(?i)#\+END_SRC\h*$ fill meta
+
+# Comment
+add-highlighter shared/org/comment region (^|\h)\K#[^+] $ fill comment
+
+# Table
+add-highlighter shared/org/table region ^\h*[|][^+] $ fill string
+
+# Various blocks
+evaluate-commands %sh{
+    blocks="EXAMPLE QUOTE EXPORT CENTER VERSE"
+    for block in ${blocks}; do
+        printf "%s\n" "add-highlighter shared/org/${block} region '(?i)#\+BEGIN_${block}\b' '(?i)#\+END_${block}\b' regions"
+        printf "%s\n" "add-highlighter shared/org/${block}/ default-region fill meta"
+        printf "%s\n" "add-highlighter shared/org/${block}/inner region \A#\+(?i)BEGIN_${block}\b\K '(?i)(?=#\+END_${block})' fill mono"
+    done
+}
+
+
+# Small example block
+add-highlighter shared/org/inline/text/example regex '^\h*[:]\h+[^\n]*' 0:meta
+
+# Unordered list items start with `-', `+', or `*'
+add-highlighter shared/org/inline/text/unordered-lists regex '^\h*([-+])\h+' 1:bullet
+
+# But `*' list must be indented with at least single space, if not it is treated as a heading
+add-highlighter shared/org/inline/text/star-list regex '^\h+([*])\h+' 1:bullet
+
+# Ordered list items start with a numeral followed by either a period or a right parenthesis, such as `1.' or `1)'
+add-highlighter shared/org/inline/text/ordered-lists regex '^\h*(\d+[.)])\h+' 1:bullet
+
+# Headings. Also includes highlighting groups for TODO and PRIORITIES
+# format: STARS TODO PRIORITY TEXT TAGS
+add-highlighter shared/org/inline/text/heading          regex   '^(?:[*]{1}|[*]{5}|[*]{9})\h+[^\n]*?(:[^:\n]*?:)?\n' 0:org_heading       1:module
+add-highlighter shared/org/inline/text/section          regex  '^(?:[*]{2}|[*]{6}|[*]{10})\h+[^\n]*?(:[^:\n]*?:)?\n' 0:org_section       1:module
+add-highlighter shared/org/inline/text/subsection       regex  '^(?:[*]{3}|[*]{7}|[*]{11})\h+[^\n]*?(:[^:\n]*?:)?\n' 0:org_subsection    1:module
+add-highlighter shared/org/inline/text/subsubsection    regex '^(?:[*]{4}|[*]{8}|[*]{12,})\h+[^\n]*?(:[^:\n]*?:)?\n' 0:org_subsubsection 1:module
+add-highlighter shared/org/inline/text/heading_comment  regex '^[*]+\h+[^\n]*((?<=\h)COMMENT)\s'                     1:comment
+add-highlighter shared/org/inline/text/heading_todo     dynregex '^[*]+\h+(?:%opt{org_todo})'                        1:org_todo 2:org_done
+add-highlighter shared/org/inline/text/heading_priority dynregex '^[*]+\h+[^\n\[]*(\[#(?:%opt{org_priority})\])'     1:org_priority
+# (?:(?:%opt{org_todo})\h+)?(\[#(?:%opt{org_priority})\])?
+# Options
+add-highlighter shared/org/inline/text/option     regex "(?i)#\+[a-z]\w*\b[^\n]*"          0:module
+add-highlighter shared/org/inline/text/title      regex "(?i)#\+title:([^\n]+)"            0:module 1:title
+add-highlighter shared/org/inline/text/requisites regex "(?i)#\+(?:author|email):([^\n]+)" 0:module 1:keyword
+
+# Drawer
+add-highlighter shared/org/inline/text/drawer   regex "^\h*([:][^\s][^\n]*?[^\s]*?[:])\W" 1:keyword
```

Comment:
`[:]` could probably be written simply as `:`, and `[^\s]` as `\S`. Note that `\n` matches `\s` too.